sem_covid/entrypoints/etl_dags/policy_watch_covid19.py

```diff
@@ -16,9 +16,9 @@
 from tika import parser
 
 from sem_covid import config
-from sem_covid.adapters.es_adapter import ESAdapter
-from sem_covid.adapters.minio_adapter import MinioAdapter
+from sem_covid.adapters.abstract_store import ObjectStoreABC
```

Comment:
this module (policy_watch_covid19.py) no longer exists. It was deleted. 
Please pull the latest changes from the main and simply made the same projections with new adapters like here.